cmd/gather.go

```diff
@@ -35,13 +36,26 @@ var gatherCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
+		if argStatic && argDynamic {
+			log.Fatalf("Cannot use both 'static' and 'dynamic' only gatherers - exiting")
+			os.Exit(1)
+		}
+
+		gathererTypeArg := ""
+		if argStatic {
+			gathererTypeArg = "static"
```

Comment:
You can actually use the consts you defined here rather than using a magic string (i.e. replace the string "static" and "dynamic" with your consts)